data/human/human missions.txt

```diff
@@ -7453,3 +7453,1280 @@ mission "Adelita 5"
+				accept
+
+	on visit
+		dialog `You land on <planet>, but realize that Kormak and Tavish are on one of your escorts who hasn't entered the system yet. Better depart and wait for them.`
+	on complete
+		conversation
+			`	The foam surf crashes against the island only a few short meters from where your ship sets down. The thundering waves make it hard to hear anything as you step outside. If you're going to have any hope of finding Hans Fuller and the stolen equipment your best bet will be the spaceport. Hopefully the sea will be a little quieter there.`
+
+
+mission "Tavish and Kormak 3"
+	name "Board the Broken Record"
+	description "Find the <npc>, disable it, board it, and retrieve the stolen scientific equipment."
+	source "Deadman's Cove"
+	waypoint "Persian"
+	destination "Hephaestus"
+	passengers 2
+	to offer
+		has "Tavish and Kormak 2: done"
+	on offer
+		conversation
+			`Hospitality is in short supply as most of the pirates and free traders you pass either ignore you entirely or give you menacing glares. The spaceport bar on the archipelago's main island functions as a sort of catchall for services you'd expect at a larger and better organized spaceport. There's various fixers, vendors, mercenaries, and opportunists plying their trades in corners, booths and tables.`
+			`	Tavish makes a beeline for one of the bartenders, the closest thing to any sort of identifiable administrator, and waves him down.`
+			`	"What'll it be, your lordship?" the bartender asks mockingly after seeing Tavish's overly formal getup.`
+			`	"I'll take a sidecar, my good man, and my large companion will have a Scottish Ale, assuming you have any." Tavish says brightly.`
+			`	The bartender pours two glasses of rum and pushes them to Tavish. "My apologies, me lord," he gives a ridiculous bow, "but we seem to be out of your favored swill."`
+			`	The bartender flashes Tavish a yellow grin that's missing a few key teeth. "Anything else I can getchya?"`
+			`	"Information." Kormak grumbles, stepping closer to the bartender, who despite Kormak's size does not flinch.`
+			`	Instead the bartender's smile only widens to give everyone a better look at his bloody gums, "Now I's might just know a thing or two. But it'll cost you a sight more than a couple glasses of rum."`
+			`	Tavish holds up his glass and looks in it, as if deciding if he's willing to drink it or not. "We need to find a ship that frequents this port of call."`
+			`	The bartender taps his overgrown moldy nails on the bar, "Ah yes, me lord. I can probably do that. For a mere 50,000 credits, that is."`
+			`	Tavish flinches at the number and then turns to look at you, "Your call."`
+			label options
+			choice
+				`	(Have Tavish attempt to talk the barkeep into giving you the information.)`
+					goto tav1
+				`	(Have Tavish attempt to talk the barkeep into giving you the information.)`
+					goto tav2
+					to display
+						has "kormak1"
```

Comment:
Why is this repeated?

Reply:
I am not sure I understand the question.

Reply:
You have the same choice in here twice, and so they will both display if you have the condition `kormak1`